scripts/lib/libvirt/libvirt_helpers.py

```diff
@@ -0,0 +1,21 @@
+import subprocess
+import os
+
+def readfile(fname):
+    with open(os.path.join(os.path.dirname(__file__), fname)) as f:
+        ret = f.read()
+    return ret
+
```

Comment:
Also this rule applies to `import` and the first code line.  But I guess tha `flake8` will yell